src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">=7.1.3",
         "symfony/error-handler": "^4.4",
         "symfony/event-dispatcher": "^4.4",
+        "symfony/http-client-contracts": "^4.4",
```

Comment:
Constraint is wrong here, should be `^1.1|^2`

Reply:
Fixed with 1cc8fc2.